sdk/proto/provider.proto

```diff
@@ -106,10 +106,6 @@ message ConfigureErrorMissingKeys {
 message InvokeRequest {
     string tok = 1;                  // the function token to invoke.
     google.protobuf.Struct args = 2; // the arguments for the function invocation.
-    string provider = 3;             // an optional reference to the provider version to use for this invoke.
-    string version = 4;              // the version of the provider to use when servicing this request.
-    bool acceptResources = 5;        // when true operations should return resource references as strongly typed.
-    string pluginDownloadURL = 6;    // an optional reference to the provider url to use for this invoke.
```

Comment:
Add a comment here that we've removed fields 3-6, and add a [reserved](https://developers.google.com/protocol-buffers/docs/proto3#reserved) line like `reserved "provider", "version", "acceptResources", "pluginDownloadURL"`? We may be able to just use `3-6`, too, but to be safe reserving the names will keep those fields from being read by an old plugin.